base/src/lib/export-schema.ts

```diff
@@ -213,7 +213,17 @@ async function exportToTypescript(
           break
         }
       }
+      case "string": {
+        const containsEnums: boolean = definition?.enum
+        if (containsEnums) {
+          const extractedEnums = definition?.enum.map(item => (item))
```

Comment:
Any reason you're mapping the string back to a string again?

Reply:
I thinks its not necessary. ill remove it